app/templates/record_edit.html

```diff
@@ -224,7 +231,7 @@ <h2>People</h2>
 
     header = $('#record_header');
     if ( $.isEmptyObject(recData) ) {
-        header.text('Add new item');
+        header.text('Add new item (An “item” is a section of text within the larger document — for example, an advertisement within a newspaper)');
```

Comment:
This is incorrect; an article or advertisement in a newspaper is itself a "document". A document now corresponds to "a cite-able unit"; ie, a citation. This is necessary for Zotero integration, and helps refine the data model. The mongo model spread citation information across multiple entities; the new data model contains all citation information at the Document level. (In fact, the database table is now called "citations".) 

An item, or whatever we're calling it these days, corresponds most closely to what Elli calls an "event" or a "factoid." In the database, these are called "references". It's a unit of information within the citation. What exactly that is, is determined by the editor, and whatever directives people want to give to the editors. Some clues, from the data model: an item (reference) is associated with multiple people in multiple roles; but one and only one date, and one and only one location. If you're looking at multiple dates or multiple locations within the same document (citation), you're looking at multiple items (references).

Reply:
In short, pick a different example. "Newspaper article" and its relatives -- "Advertisement", "Story", "Journal Article" etc. -- are now listed as document types on the document page. They're also listed as Zotero types in the zotero interface.